documentation/book/kafka-exporter/con-kafka-exporter-alerts.adoc

```diff
@@ -0,0 +1,28 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-kafka-exporter.adoc
+[id='con-metrics-kafka-exporter-alerts{context}']
+
+= Kafka Exporter alerting rule examples
+
+If you performed the steps to introduce metrics to your deployment, you will already have your Kafka cluster configured to use the alert notification rules that support Kafka Exporter.
+
+The rules for Kafka Exporter are defined in `prometheus-rules.yaml`, and are deployed with Prometheus.
+For more information, see  xref:assembly-metrics-prometheus-{context}[Prometheus].
+
+The alert notification rules specific to Kafka Exporter are as follows:
```

Comment:
I think we should add that these rules need to be adapted to correspond to the user use-cases, scenarios, etc.

Reply:
Added: 'Adapt the default configuration of these rules according to your specific needs.'